### DISCUSSION TITLE
Building my Own Spiral 12

### DISCUSSION THREAD

**Discussion by methanoid** [SpiralLinux/SpiralLinux-project#294] [Q&A]
I've fixed all references to Bullseye with Bookworm and removed a few packages from the install list that dont exist in Bookworm (deprecated) BUT I still struggle with this message on Build

The following packages have unmet dependencies:
 calamares : Depends: libboost-python1.74.0-py39 but it is not installable
             Depends: libkpmcore10 (>= 20.12.2) but it is not installable
             Depends: libpython3.9 (>= 3.9.1) but it is not installable
             Depends: libyaml-cpp0.6 (>= 0.6.2) but it is not installable

I've enabled all repos and all 4 of these are NOT in Bookworm but the version of Calamares seems to want them (but they are NOT dependencies of the Bookworm calamares but of the Bullseye calamares).

Any idea how to fix this? I think it's all I have left to fix now

**Comment by CicadaSeventeen** [comment]
You can try my fork. See https://debianeasy.github.io

**Reply by methanoid** [reply]
Thanks, you've mentioned it before....

FYI some feedback on it... A bit critical but maybe you can improve it?  I am having another play with it (and remove some of your improvements for my own use)

- I guess English isn't your first language?
- Readmes - seem absent everywhere and the repo is not userfriendly in working out what everything is - labelling could be better - it just makes repo look like it's only designed for you and nobody else is expected to use it
- Why the 1GB of extra firmware files????
- Numerous spelling mistakes/typos eg Debain everywhere - attention to detail. Not seen any in Spiral so you added these :-(
- References to Zwopper pic still included but broken
- A weird reference to "apt -y purge exim4-base nfs-common rpcbind nvidia-tesla-470-kernel-support nvidia-tesla-470-alternative nvidia-modprobe nvidia-kernel-common nvidia-installer-cleanup glx-alternative-nvidia raspi-firmware" when as far as I can see those packages are never installed. I think this is a mistake maybe?
